docs/console/game/localization/localization_key/index.md

```diff
@@ -0,0 +1,36 @@
+# Localization Keys
+
+This page lists all localization keys for the selected game
+
+## Contents
+- [Table Listing](#table-listing)
+- [Create Localization Key](#create-localization-key)
+- [Edit Localization Key](#edit-localization-key)
+- [Show/Hide Localization Key](#show-hide-localization-key)
+
+
+### Table Listing
+The Table lists the following data for each Localization Key that exists :
+- **Key** : Localization Key (which is unique)
+- **Description** : Localization Key Description field, that is optional
+- **Tags** : Different tags used for grouping keys and related
+
+### Create Localization Key
+To create a new Localization Key for a game:
+1. Click on button **Create Localization Key**
+2. On the popup, fill in **Key** field with a unique name 
+3. On the popup, optionally, fill in **Description** field with a meaningful description
+4. On the popup, fill in **tag** field with one or more tags for better categorizing
+5. Press **Save** to reflect the changes
+
+### Edit Localization Key
+To edit an existing Localization Key for a game:
+1. On the row from the key you want to edit click the button **Edit Localization Key** ![pencil](https://github.com/azerion/gamedock-sdk/raw/master/docs/console/_images/pencil.png)
+2. You can edit the values for the Localization Key as explained on the Create Localization Key section
+
+### Show/Hide Localization Key
+Hiding a Localization Key will make it not being displayed on the list of Localization Values and not being served when the Locale is published.
```

Comment:
Hiding a localization key will hide it one the list of Localization Values and not served when the locale is published